solutions/cf-231A.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: cf-231A
+source: CF
+title: Team
+author: Sathvik Chundru
+---
+
+[Official Analysis (C++)](https://codeforces.com/blog/entry/5486)
+
+## Solution (Greedy)
+
+Let's consider the following test case:
+
+```
+3
+1 1 0
+1 1 1
+1 0 0
+```
+
+The simplest solution is to input three integers per row and check 
+if their sum is greater than or equal to two. 
+
+## Implementation
+
+<LanguageSection>
```

Comment:
can you add sans's py sol from his comment on the other pr?

Reply:
lmao that was a one-liner and pretty hard to read here's a more readable one
```py
sols = 0
for _ in range(int(input())):
	sols += sum(int(i) for i in input().spllit()) >= 2
print(sols)
```
not tested but it should work

Reply:
um... ok ill add it but it looks very sus to read...

Reply:
better than that one-liner i posted earlier

Reply:
@sathu-beep can you test it to make sure it works?

Reply:
it works

Reply:
i added it now @maggie-j-liu 
merge if no more revisions needed